pyquil/paulis.py

```diff
@@ -97,6 +97,22 @@ def __len__(self):
         """
         return len(self._ops)
 
+    def copy(self):
```

Comment:
We need to document the mutability and copy behavior of the `PauliTerms` in the docs.